native/cocos/core/scene-graph/Node.cpp

```diff
@@ -600,8 +600,8 @@ void Node::setWorldScale(float x, float y, float z) {
         // convert to Matrix 3 x 3
         Mat3::fromMat4(tmpLocalTransform, &localRS);
         Mat3::fromQuat(_localRotation.getConjugated(), &localRotInv);
-        // remove rotation part of the local matrix 
-        Mat3::multiply(localRS, localRotInv, &localRS);
+        // remove rotation part of the local matrix
+        Mat3::multiply(localRotInv, localRS, &localRS);
```

Comment:
Why change it?

Reply:
![1669969287263](https://user-images.githubusercontent.com/40414978/205248372-31de5a79-b966-4a2c-92b0-cbddd7951c75.png)

Sync with the logic in ts